themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Announcing Preview for Helm Release Resource for Kubernetes
+h1: "Announcing Preview for Helm Release Resource for Kubernetes"
+date: "2021-09-07"
+meta_desc: "Today we are announcing preview of a new Helm Release resource for Kubernetes which adds support for Helm natively on Pulumi."
+meta_image: "helm-release.gif"
```

Comment:
Not sure this can be a GIF. Would be good to confirm with Christian and also confirm, if it can be a GIF, that the GIF is the correct size.

Reply:
> Not sure this can be a GIF. Would be good to confirm with Christian and also confirm, if it can be a GIF, that the GIF is the correct size.

The gif does render and I have seen it in some of the older Kubernetes posts. But it definitely is pretty illegible in the index view. 
@cnunciato do you have any recommendations? I am happy to switch to a generic Pulumi image like https://www.pulumi.com/blog/kubernetes-fundamentals-part-one/k8s-fundamentals.png instead.

Reply:
Yeah, unfortunately the preview/meta image shouldn't be an animated GIF because social media previews won't render them consistently. See here, for example, with the [Twitter card validator](https://cards-dev.twitter.com/validator), which just shows the first frame, I believe: 

![image](https://user-images.githubusercontent.com/274700/132548175-4f9d191a-2d99-4533-bf84-5dbd0108a8e5.png)

I'd suggest going with a bigger PNG for the meta image (I put some [recommended specs](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md#media) in BLOGGING.md) and maybe finding a place for the animated GIF in the body of the post.

Reply:
I have added a separate image - apologies in advance if it looks hideous. I used one of the sanctioned logo images from [Helm](https://github.com/cncf/artwork/blob/master/examples/graduated.md#helm-logos) on an existing k8s meta image.